lib/mayaUsd/ufe/UsdUndoInsertChildCommand.cpp

```diff
@@ -28,9 +28,15 @@
 #include <pxr/usd/usd/stage.h>
 #include <pxr/usd/usdGeom/gprim.h>
 
+#include <maya/MGlobal.h>
+#include <maya/MItSelectionList.h>
+#include <maya/MSelectionList.h>
```

Comment:
Do we need any of these?

Reply:
@ppt-adsk Ah sorry... I was experimenting with something and forgot to remove them. 

Fixed: cc3a604fa3d934e585d899813257eecd45702a8d